components/skus/browser/rust/wasm/Makefile

```diff
@@ -0,0 +1,7 @@
+all: build
+	
+
+.PHONY: build
+build: 
+	cargo build --lib --release --target wasm32-unknown-unknown
```

Comment:
how do these build in gn?

Reply:
they don't, do they need to? the resulting wasm lib isn't used by the browser itself

Reply:
question answered in DM - we don't use this in Brave